ProjectName/src/screens/PaymentDetails/PaymentDetailsForm/payment-details-form.styles.ts

```diff
@@ -0,0 +1,40 @@
+import {StyleSheet} from 'react-native';
+export const styles = StyleSheet.create({
+  buttonContainer: {
+    marginTop: 22,
+  },
+  container: {
+    backgroundColor: 'white',
+    flexGrow: 1,
+    paddingHorizontal: 16,
+  },
+  inputBox1: {
```

Comment:
If it's possible, please try to avoid using numbers in naming things, unless it's really neseccary or clear.
From my experience numbers gives a clue of some ordering. But order could be changed. Imagine we would be forced to add some input between 1 and 2. Now our naming approach makes it's not that easy to adjust code. This would ignite a chain `more renaming => more code change => more bugs`.
Try to use some abstract naming, if possible.